packages/2020_Bongers_SouthPeru/.~lock.2020_Bongers_Inc.ssf#

```diff
@@ -0,0 +1 @@
+,dhananjaya_aththanayaka,bionb53,21.02.2024 22:56,file:///home/dhananjaya_aththanayaka/.config/libreoffice/4;
```

Comment:
@93Boy please check the "Files changed" tab, then you can easily catch weird libra-office files like this one. Please remove.

I will leave it to @TCLamnidis to review the SSF file.